### PR TITLE
getTranslationDomain() $translation variable name typo fixed

### PR DESCRIPTION
**Pull request by stas-p**
Fixes the "Call to a member function getConfig() on null" error

### CODE REVIEW

File: Services/FormErrorsParser.php [hunk 1]
```diff
@@ -69,7 +69,11 @@ private function realParseErrors(FormInterface $form, array &$results)
             if (empty($label)) {
                 $label = ucfirst(trim(strtolower(preg_replace(['/([A-Z])/', '/[_\s]+/'], ['_$1', ' '], $name))));
             }
-            $results[] = array('name' => $name, 'label' => $label, 'errors' => $errors);
+            $errorMessages = array();
+            foreach ($errors as $error) {
+                $errorMessages[] = $error;
+            }
```
Comment (Ex3v) [inline Services/FormErrorsParser.php]: Could you explain why are you rewriting `$errors` table inside of this loop?

**Review by Ex3v — CHANGES_REQUESTED** [review]

**Comment by Ex3v** [comment]
In general - thanks a lot! Let's just clear this one small thing out and it will be merged

**Comment by spectrals** [comment]
Maciej, thanks for putting this bundle together. With some minor changes I found it pretty useful when building errors list for the API I'm working on.

I did few more commits recently - you may want to review them:
- Fixed deprecated twig extension declaration (Twig 2.0+ compatibility)
- Added handling for the case when form parent is null (it can be according to the phpdoc annotations)
- Translator is now moved to the parser service instead of a twig extension, label translation is now handled properly
- Netbeans project folders were removed (let me sure if there was a reason for them to be there :) )

**Comment by Ex3v** [comment]
Oh man, that's what makes me happy - when someone says that I made job easier by this little helpful thing. Thanks!
The NetBeans thing has a simple explanation - that was my first bundle ever and I wasn't sure what I was doing ;) Thanks a lot for removing those. 
One thing I still miss as for today is lack of unit tests. I'm gonna write them one day.
I will try to take a look at all of your changes when I get myself to the office. Thanks once more!

**Review by Ex3v — APPROVED** [review]

**Comment by Ex3v** [comment]
@spectrals could you please resolve conficts with `Twig/FormErrorsExtension.php`? Everything else looks great

**Comment by spectrals** [comment]
Done! Thanks for reviewing these so quickly

**Pull request merged**